nanoc/test/checking/checks/test_internal_links.rb

```diff
@@ -111,8 +125,22 @@ def test_unescape_url
 
       check = Nanoc::Checking::Checks::InternalLinks.create(site)
 
-      assert check.send(:valid?, 'stuff/right%20foo', 'output/origin')
-      refute check.send(:valid?, 'stuff/wrong%20foo', 'output/origin')
+      assert check.send(:valid?, path_to_file_uri('stuff/right%20foo', site), 'output/origin')
+      refute check.send(:valid?, path_to_file_uri('stuff/wrong%20foo', site), 'output/origin')
+    end
+  end
+
+  def deeply_nesten_relatives
```

Comment:
Same here: add `test_` prefix

(I eventually want to move fully to RSpec, but it’s a slow work in progress.)